src/components/Personalization/index.js

```diff
@@ -93,6 +114,38 @@ const createPersonalization = ({ config, logger, eventManager }) => {
 
         collectClicks(merger, clickedElement, storage);
       }
+    },
+
+    commands: {
+      getDecisions(options) {
+        let { xdm } = options;
+        const {
+          scopes = [],
+          data,
+          type: eventType,
+          mergeId: eventMergeId
+        } = options;
+        const event = eventManager.createEvent();
+
+        if (eventType || eventMergeId) {
+          xdm = Object(xdm);
+        }
+
+        if (eventType) {
+          assign(xdm, { eventType });
+        }
+
+        if (eventMergeId) {
+          assign(xdm, { eventMergeId });
+        }
+
+        event.setUserXdm(xdm);
+        event.setUserData(data);
+
+        validateEvent(event, logger);
+
+        return eventManager.sendEvent(event, { scopes });
```

Comment:
I believe this would be exposing the full response to the user, which would be too much for the customer to have to sort through and too much public API we would need to maintain. We should sort through the response ourselves, gather the decisions in whatever structure the customer would find most useful, then resolve the promise we give the user with the nicely packaged decision information.

Reply:
yes, we'll process and package the response that we'll get back to the customer in a nicely format.